### ISSUE TITLE
Sizes out of whack

### ISSUE THREAD

**Issue by MadAsKiwi** [jordan-thoms/2Degrees-Toolbox#8]
Hi Jordan

Firstly, thanks for the awesome app.  It is way better than the 2dm offering.

I do have one wee issue at the moment though.  I have just installed your app on my wife's phone, and found that the size offerings were 3x2 and 3x3, and no options for the 1x2 and the 2x2 that we used to have.  I ran the update on my phone, and the widget screen shows the 3x2 and 3x3, and the old 1x2 widget on my home screen is all squished up.

Is this an issue/feature that will be sorted in your next update?

Please see the attached screen shots of the issue on my phone.

The phones we are using are both Samsung Galaxy SIII Mini's.

Many thanks

Rob
![screenshot_2013-09-26-19-01-17](https://f.cloud.github.com/assets/5547362/1215928/777a1fb4-267a-11e3-92e2-9125660b2636.png)
![screenshot_2013-09-26-19-01-30](https://f.cloud.github.com/assets/5547362/1215929/83517832-267a-11e3-9753-e19d8d8d2085.png)

**Comment by paulbrou** [comment]
Same issue for me. Version 1.5.9.1, on a Nexus 4 running 4.3.

![img_20131115_205236](https://f.cloud.github.com/assets/4108366/1548505/8e06d75e-4dcb-11e3-9d6e-06cb8696192e.JPG)

**Comment by phoglenix** [comment]
Same issue for me. Version 1.5.9.1, on a Nexus 5 running Cyanogenmod11 M8.
In CM you can manually resize widgets. I made mine 3x2 and now it won't go back to being 2x2 or 1x2

**Comment by jordan-thoms** [comment]
Yeah, the widgets are messed up. It's because I updated the target sdk without making the change to widget sizes to remove the borders. I'm working on a new version with a single resizable widget but progress is slow :Z

**Comment by MadAsKiwi** [comment]
Cool.. watching with interest, Jordan.  Thanks for the update.

**Comment by MadAsKiwi** [comment]
Hi Jordan.  I have just done the update now, and the app is now able to get my information.  Thanks for that.  Also, thanks for sorting the size issue.  However, I still find that the 1x2 box is still squashed up like in my original screen shot.  I will flick you an email regarding beta testing also.  Thanks for your assistance with this.

**Comment by jordan-thoms** [comment]
Yeah, the widgets still aren't great - I'm going to work on getting a better setup for the new version of the app.

**Comment by phoglenix** [comment]
Thanks for the quick update! Working correctly again for me :)
